src/Features/Lsif/Generator/Program.cs

```diff
@@ -86,14 +89,23 @@ public static Task Main(string[] args)
                 outputWriter = outputFile;
             }
 
-            using var logFile = log != null ? new StreamWriter(log) { AutoFlush = true } : TextWriter.Null;
             ILsifJsonWriter lsifWriter = outputFormat switch
             {
                 LsifFormat.Json => new JsonModeLsifJsonWriter(outputWriter),
                 LsifFormat.Line => new LineModeLsifJsonWriter(outputWriter),
                 _ => throw new NotImplementedException()
             };
 
+            using var logFile = logFileName is not null and not "stderr" ? new StreamWriter(logFileName) { AutoFlush = true } : null;
+
+            ILogger logger;
+            if (logFile is not null)
+                logger = new PlainTextLogger(logFile);
```

Comment:
do we never want to write the lsif format log to a file?

Reply:
@gundermanc do you want that?

Reply:
If the standard error stream works, I don't intend on using the log file.